### ISSUE TITLE
Can't build Repo: TSError: ⨯ Unable to compile TypeScript

### ISSUE THREAD

**Issue by ReevMich** [angular/material2#8038]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Material 2 produces a release build.

#### What is the current behavior?
I'm new to the angular world and I'm trying to find a way to contribute to the project but when I follow the development steps from the DEVELOPMENT.md to build the project, I get the following error.

```
C:\Users\cooly\Desktop\material2-master>gulp material:build-release

C:\Users\cooly\Desktop\material2-master\node_modules\ts-node\src\index.ts:307
        throw new TSError(formatDiagnostics(diagnosticList, cwd, ts, lineOffset))
              ^
TSError: ⨯ Unable to compile TypeScript
tools\gulp\tasks\lint.ts (90,24): Property 'red' does not exist on type 'typeof "C:/Users/cooly/Desktop/material2-master/node_modules/@types/gulp-util/node_modules/chalk/...'. (2339)
    at getOutput (C:\Users\cooly\Desktop\material2-master\node_modules\ts-node\src\index.ts:307:15)
    at C:\Users\cooly\Desktop\material2-master\node_modules\ts-node\src\index.ts:336:16
    at Object.compile (C:\Users\cooly\Desktop\material2-master\node_modules\ts-node\src\index.ts:498:11)
    at Module.m._compile (C:\Users\cooly\Desktop\material2-master\node_modules\ts-node\src\index.ts:392:43)
    at Module._extensions..js (module.js:579:10)
    at Object.require.extensions.(anonymous function) [as .ts] (C:\Users\cooly\Desktop\material2-master\node_modules\ts-node\src\index.ts:395:12)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
```
#### What are the steps to reproduce?
1. I just downloaded the repo from GitHub
2. Unzipped the project
3. Followed the steps from the DEVELOPMENT.md to try to build the project.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Material 2 beta
Angular: Repo Angular Version
OS: Windows 10
Typescript: 2.4.2

**Comment by willshowell** [comment]
Likely related to https://github.com/angular/material2/pull/8030

**Comment by crisbeto** [comment]
Will be resolved by #8030.
